tests/openshift4.yml

```diff
@@ -0,0 +1,2 @@
+parameters:
+  resource_locker: {}
```

Comment:
Why is this called openshift4 if it actually isn't openshift4?

Reply:
Because I forgot to add the fact, obviously

Reply:
I think you just swapped default and openshift4 right?

Reply:
actually the test cases were swapped, now `defaults` tests non-OCP4 and `openshift4 tests OCP4